### ISSUE TITLE
How to configure a VB vm, without attempting a boot since PXE boot server is not available at that time?

### ISSUE THREAD

**Issue by strus38** [hashicorp/vagrant#11927]
Hi
I am deploying a set of VMs using Vagrant on VirtualBox, some VMs are setup to boot on the network, but the server is not ready when those VMs are defined, so is there a way for Vagrant, to only configure the VM, but do not attempt a boot?
I did not find anything in the doc, but I may have bad eyes.

Thanks

**Comment by soapy1** [comment]
🤔 I'm not sure if what is being described here is possible currently. 
I tried this out locally with the virtualbox provider. I did an initial `vagrant up` for my VM to that Vagrant could have knowledge of it. Then I was able to do some operations outside of Vagrant like powercycle it and what have you. Regardless of the final state I left the VM in after doing operations out of Vagrant `vagrant up` seemed to produce the right result. So, maybe this is do-able if you are able to do an initial Vagrant boot?
